docs/guides/azure-federation.md

```diff
@@ -0,0 +1,138 @@
+# Federating Coder's control plane to Azure
+
+<div>
+  <a href="https://github.com/ericpaulsen" style="text-decoration: none; color: inherit;">
+    <span style="vertical-align:middle;">Eric Paulsen</span>
+    <img src="https://github.com/ericpaulsen.png" width="24px" height="24px" style="vertical-align:middle; margin: 0px;"/>
+  </a>
+</div>
+January 26, 2024
+
+---
+
+This guide will walkthrough how to authenticate Coder's Terraform runner to
```

Comment:
```suggestion
This guide will walkthrough how to authenticate Coder's Terraform runner to
```
```suggestion
This guide will walkthrough how to authenticate a Coder Provisioner to
```
Should we use the term Coder Provisioner instead of Runner?